src/lxc/af_unix.c

```diff
@@ -56,7 +56,7 @@ int lxc_abstract_unix_open(const char *path, int type, int flags)
 	addr.sun_family = AF_UNIX;
 
 	len = strlen(&path[1]) + 1;
-	if (len >= sizeof(addr.sun_path) - 1) {
```

Comment:
if we use a long enough lxcpath, in the code `len = strlen(&path[1]) + 1` , len = 107. and `sizeof(addr.sun_path)-1` = 107.  at this situation, the abstract unix socket length is legal. we can fix it to `len > sizeof(addr.sun_path) - 1` or `len >= sizeof(addr.sun_path) `